### PR TITLE
Refactor internet URL logic out

### PR DESCRIPTION
**Pull request by inverse**
Saw that CI was failing so attempted to fix it.

Refactored logic for internet URL generation into common place

### CODE REVIEW

File: src/src/Factory/TaskFactory.php [hunk 1]
```diff
@@ -165,39 +160,17 @@ public function createSetEdgeappOptionsTask(string $id, array $options): Task
 
     public function createEnableOnlineTask(string $id): Task
     {
-        $domain_option = $this->optionRepository->findOneBy(['name' => 'DOMAIN_NAME']);
```
Comment (inverse) [inline src/src/Factory/TaskFactory.php:168]: Moved the majority of this into the getInternetUrl

File: src/src/Helper/SystemHelper.php [hunk 2]
```diff
@@ -32,14 +32,15 @@ public function getUptimeInSeconds(): int
...
+    public function isCloud(): bool
+    {
+        return self::VERSION_CLOUD === $this->getReleaseVersion();
+    }
+
     public function getReleaseVersion(): ?string
     {
         $release_version_option = $this->optionRepository->findOneBy(['name' => 'RELEASE_VERSION']) ?? new Option();
 
-        if (null === $release_version_option->getValue()) {
```
Comment (inverse) [inline src/src/Helper/SystemHelper.php:39]: Same as without it :)

File: src/src/Repository/OptionRepository.php [hunk 3]
```diff
@@ -19,6 +19,32 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, Option::class);
     }
 
+    public function findDomainName(): ?string
```
Comment (inverse) [inline src/src/Repository/OptionRepository.php:22]: Util functions for getting at config in a nicer way - will be easier to mock too :+1:

File: src/src/Helper/EdgeAppsHelper.php [hunk 4]
```diff
@@ -51,27 +52,20 @@ public function edgeAppExists(string $app_id): bool
...
-        //     // Check if registration was successfull and only then issue the appliance to set configurations.
-        //     if (!empty($app_info) && !empty($app_info['url'])) {
-        //         $url = $app_info['url'];
-        //     }
-        // }
+            return sprintf('%s-%s.%s', $host, $appId, $cluster);
+        }
 
-        return $url;
+        return null;
```
Comment (inverse) [inline src/src/Helper/EdgeAppsHelper.php:69]: if not cloud will be `null` is this expected?

**Review by paulotruta — APPROVED** [review]

**Pull request merged**